src/AppBundle/Resources/views/themes/app/user/activate.html.twig

```diff
@@ -5,7 +5,7 @@
 {% form_theme form '@ezdesign/forms/theme.html.twig' %}
 
 {% block content %}
-    <div class="page-header">
```

Comment:
Even though this css class has been deprecated in B4, it should be left in because of the possible styling of the main page title